index.html

```diff
@@ -52,6 +52,183 @@ <h2 class="contact__title">LET'S CONNECT</h2>
+                            <h4 class="card__canopy-title">CANOPY</h4>
+                            <ul class="card__canopy-list">
+                                <li>Backed Dev</li>
+                                <li>2023</li>
+                            </ul>
+                        </div>
+                        <p class="card__canopy-text">endpoints for a vending machine that is
+                            accessible to buyers and sellers.(this project is private, get contact with me)</p>
+                    </div>
+                    <ul class="card__list">
+                        <li>Node.js</li>
+                        <li>JavaScript</li>
+                        <li>Redis</li>
+                        <li>Docker</li>
+                    </ul>
+                    <a href="mailto:ahmedeid2684@gmail.com" class="btn btn--green">See Project</a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about">
+        <h2 class="about__title">About <br />
+            Myself</h2>
+        <p class="about_description">
+            I can help you build a product, feature, or website. Check out my portfolio and experience. If you like what
+            you see and have a project you need coded, contact me without hesitation.
+        </p>
+        <h2 class="about_contact__title">LET'S CONNECT</h2>
+        <ul class="about_portfolio__icons">
+            <li>
+                <a href="https://twitter.com/ahmedeid2684" rel="noopener noreferrer" target="_blank">
+                    <img src="images/twitter.svg" alt="twitter-logo">
+                </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/ahmed-eid-0018571b1/" rel="noopener noreferrer" target="_blank">
+                    <img src="images/linkedin.svg" alt="linkedin-logo">
+                </a>
+            </li>
+            <li>
+                <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
+                    <img src="images/github.svg" alt="github-logo">
+                </a>
+            </li>
+        </ul>
+        <div class="about__btn">
+            <a href="https://drive.google.com/file/d/1b3KKgW82D9qDyDKbZ8u8TfOwGGFjV8Y1/view?usp=sharing"
+                class="btn ">get me resume</a>
+        </div>
+        <ul class="about__card__container">
+
+            <li class="about__card">
+
+                <div class="about__imgbox">
+                    <img src="images/programming-language.png" alt="Languages-icon" class="about__card__img">
+                </div>
+
+                <div class="card__details">
+
+                    <h3 class="card__text about__text__card">Languages</h3>
+
+                    <ul class="about__list">
+                        <li>JavaScript</li>
+                        <li>TS</li>
+                        <li>C++</li>
+                        <li>HTML</li>
+                    </ul>
+
```

Comment:
- [ ] The same applies to your languages section. Kindly follow the design guidelines for consistency

![image](https://user-images.githubusercontent.com/98453979/235092010-8c9d2b55-549e-4ee6-807b-dade94845d61.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/235092243-c165ad18-7cc5-422a-8745-947fa40d65a7.png)